listenbrainz/db/user.py

```diff
@@ -456,6 +456,45 @@ def get_similar_users(user_id: int) -> Optional[SimilarUsers]:
         return SimilarUsers(user_id=user_id, similar_users=users)
 
 
+def get_similar_users_as_list(user_id: int) -> Tuple[List[dict], dict]:
```

Comment:
A new method shouldn't be needed for this. There is already an existing method for retrieveing similar users.

Reply:
Yup, modifying original method sounds better instead of returning a dict and a list just return the list and let the caller function convert it to a dict where required.

Reply:
Alright so that means wherever this function is called rn, I should insert the conversion there. Also old method uses SimilarUsers() class. Should I edit all of its occurrences as well?

Reply:
Yes. you'll have to edit the other occurrences as well.